tests/src/python/test_qgsvectorlayertemporalproperties.py

```diff
@@ -208,7 +214,7 @@ def testDualFieldMode(self):
         context = QgsVectorLayerTemporalContext()
         context.setLayer(layer)
 
-        range = QgsDateTimeRange(QDateTime(QDate(2019, 3, 4), QTime(11, 12, 13)), QDateTime(QDate(2020, 5, 6), QTime(8, 9, 10)))
+        range = QgsDateTimeRange(QDateTime(QDate(2019, 3, 4), QTime(11, 12, 13)), QDateTime(QDate(2020, 5, 6), QTime(8, 9, 10)), includeBeginning=False, includeEnd=False)
```

Comment:
idem. To me it would be ok to remove this test cases, but untill my PR is merged or somebody agrees with my reasoning I thought not to remove those test cases.